src/styles/bar.css

```diff
@@ -63,8 +70,21 @@
   text-indent: -9999em;
```

Comment:
Only to  push the text content outside of the viewport and keep the place for the icon. It was the best practice for a long time, but maybe @GoOz knows a better way to do this?